### PR TITLE
More things!

### PR DESCRIPTION
**Pull request by AstroOrbis**

### CODE REVIEW

File: mac-cleanup [hunk 1]
```diff
@@ -118,11 +118,14 @@ msg 'Clearing System Cache Files...'
 sudo rm -rfv /Library/Caches/* &>/dev/null
 sudo rm -rfv /System/Library/Caches/* &>/dev/null
 sudo rm -rfv ~/Library/Caches/* &>/dev/null
+sudo rm -rfv /private/var/folders/bh/5qn6f27x049drtr8rqq75jt80000gp/C/*/* &>/dev/null
```
Comment (fwartner) [inline mac-cleanup:121]: Are you sure that THIS folder will be on every machine?!

Comment (AstroOrbis) [reply mac-cleanup:121]: You're right, I should just remove the general temp file directory instead... That folder has been on my system for as long as I can remember, so I must have been biased.

**Comment by AstroOrbis** [comment]
Basically, it clears the contents of all subfolders in each letter folder of each huge folder.
I found the path with CleanMyMacX and decided to add it here :3 
It has the App Store cache, among other caches.

**Comment by necrotech** [comment]
I definitely don't want everything in ~/.cache deleted.  I'm pretty sure some other people don't, either.

**Comment by AstroOrbis** [comment]
@necrotech fixed!

**Pull request merged**